src/client.js

```diff
@@ -4,8 +4,18 @@ import 'babel-polyfill';
 import { hydrate } from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { ensureReady, After } from '@jaredpalmer/after';
+import * as OfflinePluginRuntime from 'offline-plugin/runtime';
 import routes from './app/routes';
 
+if (process.env.NODE_ENV === 'production') {
+  OfflinePluginRuntime.install({
+    // Source: https://zach.codes/handling-client-side-app-updates-with-service-workers/
+    // NB this means links must have special logic. I have no links to add this on yet, see the link above.
```

Comment:
We can use the Link here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/InlineLink/index.jsx#L6

Reply:
If updating the SW on each Link onClick is not going to be implemented in this PR, I'd suggest opening an issue to capture this work, so it is not forgotten.

Reply:
Link for SW busting https://github.com/bbc/simorgh/issues/571